sentry-core/src/scope/real.rs

```diff
@@ -234,10 +234,10 @@ impl Scope {
             }
         }
 
-        event.breadcrumbs.extend(self.breadcrumbs.iter().cloned());
-        event.extra.extend(self.extra.iter().cloned());
-        event.tags.extend(self.tags.iter().cloned());
-        event.contexts.extend(self.contexts.iter().cloned());
+        event.breadcrumbs.extend(self.breadcrumbs.clone().into_iter());
```

Comment:
Just run rustfmt on this:
```
-        event.breadcrumbs.extend(self.breadcrumbs.clone().into_iter());
+        event
+            .breadcrumbs
+            .extend(self.breadcrumbs.clone().into_iter());
```

But can you explain why you need this? Is this an API change in `im`?

Reply:
Sure thing. (Sorry, I always forget to `fmt`)

Yeah, this looks to be a change required by `im`. 
`extend()` changed from accepting a reference for a tuple to a tuple of references.

```
error[E0271]: type mismatch resolving `<im::hashmap::Iter<'_, std::string::String, sentry_types::protocol::v7::Context> as std::iter::Iterator>::Item == &_`
   --> sentry-core/src/scope/real.rs:243:52
    |
243 |         event.contexts.extend(self.contexts.iter().cloned());
    |                                                    ^^^^^^ expected tuple, found reference
    |
    = note:  expected tuple `(&std::string::String, &sentry_types::protocol::v7::Context)`
            found reference `&_`
```